examples/00-EDB/12_edb_sma_connector_on_board.py

```diff
@@ -70,9 +69,9 @@
 for layer_name in edb.stackup.signal_layers.keys():
     gnd_dict[layer_name] = edb.modeler.create_rectangle(layer_name, "GND", [0, "PCB_W/-2"], ["PCB_L", "PCB_W/2"])
 
-###################
-# Create signal net
-# ~~~~~~~~~~~~~~~~~
+###############################################################################
+# ## Create signal net
+#
 # Create signal net on layer 3, and add clearance to the ground plane.
```

Comment:
```suggestion
# Create a signal net on layer 3 and add clearance to the ground plane.
```
Should we use "on the third layer" instead of "on layer3"?